lib/sprig/reap/inputs/environment.rb

```diff
@@ -0,0 +1,24 @@
+module Sprig::Reap::Inputs
+  class Environment
+    attr_reader :input
+
+    def self.default
+      new().env
+    end
+
+    def self.parse(input)
+      new(input).env
+    end
+
+    def initialize(input = nil)
```

Comment:
Can't this just be `def initialize(input = Rails.env)`? Then you can eliminate the `||` below.